runtime/src/message_processor.rs

```diff
@@ -123,6 +124,7 @@ impl<'a> ThisInvokeContext<'a> {
             sysvars: RefCell::new(vec![]),
             blockhash,
             fee_calculator,
+            return_data: Vec::new(),
```

Comment:
"Option` instead? to avoid allocs for invoke that don't use return data?

Reply:
So it has become an option now.